src/js/view/controls/controls.js

```diff
@@ -23,6 +23,7 @@ require('css/controls.less');
 const ACTIVE_TIMEOUT = OS.mobile ? 4000 : 2000;
 // Keys which bypass keyboard shortcuts being off
 const ALWAYS_ALLOWED_KEYS = [27];
+let shouldFocusOnInactive = false;
```

Comment:
if the user clicks off the controls or never interacts with them, we want to ensure we do not falsely focus the player

Reply:
this also controls the other scenarios listed in the pr descript